runtime/master/src/main/java/edu/snu/nemo/runtime/master/scheduler/BatchScheduler.java

```diff
@@ -93,6 +93,7 @@ private BatchScheduler(final TaskDispatcher taskDispatcher,
           .subscribe(updatePhysicalPlanEventHandler.getEventClass(), updatePhysicalPlanEventHandler);
     }
     this.executorRegistry = executorRegistry;
+    this.dataSkewDynOptDataHandler = new DataSkewDynOptDataHandler();
```

Comment:
Do we need more than one handler to handle multiple RuntimePasses in the future?
Add TODO here?

Reply:
For that case, I'll refactor `DynOptDataHandler` to keep a list of data handlers and add comments on it.